{"uuid": "d0f2229d-6206-44ae-8826-19572be849ce", "children": ["66bf71b2-5112-4add-9213-2056b0d0e3b0"], "befores": [{"name": "start_appium_desired", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.WebDriverException: Message: An unknown server-side error occurred while processing the command. Original error: Could not proxy command to the remote server. Original error: socket hang up\n\n", "trace": "  File \"D:\\daima\\venv\\lib\\site-packages\\pluggy\\callers.py\", line 187, in _multicall\n    res = hook_impl.function(*args)\n  File \"D:\\daima\\venv\\lib\\site-packages\\_pytest\\fixtures.py\", line 1126, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"D:\\daima\\venv\\lib\\site-packages\\_pytest\\fixtures.py\", line 932, in call_fixture_func\n    fixture_result = fixturefunc(**kwargs)\n  File \"D:\\appiumdm\\Appium_AutoTest\\testcase\\conftest.py\", line 38, in start_appium_desired\n    driver = appium_desired_caps(host, port)\n  File \"D:\\appiumdm\\Appium_AutoTest\\base\\DesireCaps.py\", line 40, in appium_desired_caps\n    driver = webdriver.Remote('http://%s:%s/wd/hub' % (host, port), desired_caps)  # IP地址是Appium中的监听地址\n  File \"D:\\daima\\venv\\lib\\site-packages\\appium\\webdriver\\webdriver.py\", line 156, in __init__\n    AppiumConnection(command_executor, keep_alive=keep_alive), desired_capabilities, browser_profile, proxy\n  File \"D:\\daima\\venv\\lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 157, in __init__\n    self.start_session(capabilities, browser_profile)\n  File \"D:\\daima\\venv\\lib\\site-packages\\appium\\webdriver\\webdriver.py\", line 225, in start_session\n    response = self.execute(RemoteCommand.NEW_SESSION, parameters)\n  File \"D:\\daima\\venv\\lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 321, in execute\n    self.error_handler.check_response(response)\n  File \"D:\\daima\\venv\\lib\\site-packages\\selenium\\webdriver\\remote\\errorhandler.py\", line 242, in check_response\n    raise exception_class(message, screen, stacktrace)\n"}, "start": 1618234610496, "stop": 1618234623237}], "start": 1618234610496, "stop": 1618234623368}